ols/app/models/config.py

```diff
@@ -90,6 +91,21 @@ def _file_check(path: FilePath, desc: str) -> None:
         raise InvalidConfigurationError(f"{desc} '{path}' is not readable")
 
 
+def _get_log_level(value: str) -> int:
```

Comment:
-> utils/logging.py perhaps?

Reply:
cause circular import - InvalidConfigurationError in .../config.py <-> _get_log_level in .../logging.py